driver-core/src/main/com/mongodb/internal/operation/BulkWriteBatch.java

```diff
@@ -169,9 +164,8 @@ private BulkWriteBatch(final MongoNamespace namespace, final ConnectionDescripti
         if (!payloadItems.isEmpty()) {
             command.put(getCommandName(batchType), new BsonString(namespace.getCollectionName()));
             command.put("ordered", new BsonBoolean(ordered));
-            if (!writeConcern.isServerDefault() && !sessionContext.hasActiveTransaction()) {
-                command.put("writeConcern", writeConcern.asDocument());
-            }
```

Comment:
This logic needs to be reused in the implementation of the new `bulkWrite` command, so I extracted it into `MixedBulkWriteOperation.commandWriteConcern`.

Reply:
Will it eventually move to some common "helper" class?

Reply:
I had no such plans, as I don't see any value in creating a somewhat useless class just to hold this method, such that both `MixedBulkWriteOperation` and `ClientBulkWriteOperation` could call `commandWriteConcern` on that class. Calling `commandWriteConcern` on `MixedBulkWriteOperation` seems to be not worse in any way, especially given that when it is statically imported, one can't even notice immediately see where the method comes from.